From a817386f1d0539a040fbdc02853a73bbcdddf1f1 Mon Sep 17 00:00:00 2001
From: Yogesh Tyagi <yogesh.tyagi@intel.com>
Date: Fri, 30 Sep 2022 20:26:04 +0800
Subject: [PATCH] Fix build issues due to gflag and zlib

Change gflag to shared as we are not building with static library.
We are using zlib from host, so don't add zlib as third party package

Upstream-Status: Inappropriate

Signed-off-by: Yogesh Tyagi <yogesh.tyagi@intel.com>
---
 thirdparty/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/thirdparty/CMakeLists.txt b/thirdparty/CMakeLists.txt
index 4cba1588fe..f82bf3148b 100644
--- a/thirdparty/CMakeLists.txt
+++ b/thirdparty/CMakeLists.txt
@@ -39,7 +39,7 @@ if(NOT CMAKE_CROSSCOMPILING)
 endif()
 
 if(NOT zlib_FOUND)
-    add_subdirectory(zlib EXCLUDE_FROM_ALL)
+	#add_subdirectory(zlib EXCLUDE_FROM_ALL)
 endif()
 
 #
@@ -85,7 +85,7 @@ ov_install_static_lib(fluid openvino_common)
 
 if(NOT APPLE)
     # on Apple only dynamic libraries are available
-    find_package(gflags QUIET COMPONENTS nothreads_static)
+    find_package(gflags QUIET COMPONENTS nothreads_shared)
 endif()
 
 if(gflags_FOUND)
-- 
2.37.3

